wiki-thread-lighting-app.md

```diff
@@ -0,0 +1,103 @@
+## How to Set Up and Control a Matter Thread Lighting Device
+
+This guide will walk you through the process of building, running, and controlling a Matter Thread lighting application over a Thread network. 
+
+Here are step-by-step instructions:
+
+#### Hardware Setup
+- Machine A
+  - nRF52840 dongle as [Radio Co-Processor (RCP)](https://openthread.io/platforms/co-processor#radio_co-processor_rcp)
+  - Choose one of the following options:
+    - PC running Ubuntu Classic 23.04 OS
+    - Raspberry Pi 4B running Ubuntu Server 22.04 OS or Ubuntu Core 22, and an LED light
```

Comment:
> Choose one of the following options

A little complicated. Is this implying that Ubuntu Server/Desktop 23.04 on Pi isn't going to work? What about UC22 on a PC?

Maybe the Option 1 and Option 2 referenced in the instructions should also be used here.

```suggestion
  - Option 1: PC running Ubuntu Classic 23.04 OS
  - Option 2: Raspberry Pi 4B running Ubuntu Server 22.04 OS or Ubuntu Core 22, and an LED light
```